api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -2,25 +2,41 @@ syntax = "proto3";
 
 package clutch.chaos.experimentation.v1;
 
+import "api/v1/annotations.proto";
 import "google/api/annotations.proto";
 import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
 import "validate/validate.proto";
 
-import "api/v1/annotations.proto";
-
 option go_package = "experimentationv1";
 
 message Experiment {
   fixed64 id = 1;
+
+  // The experiment configuration specific to the type of experiment
   google.protobuf.Any config = 2;
+
+  // The time when the experiment should start. If unspecified, defaults to 'now'
+  google.protobuf.Timestamp start_time = 3;
+
+  // The time when the experiment should end, If unspecified, the experiment runs indefinitely unless it is manually
+  // stopped by a user.
+  google.protobuf.Timestamp end_time = 4;
```

Comment:
what about `scheduled_end_time` and `creation_time` fields? Should their addition be a part of this PR?

Reply:
i can add creation_time if you want to put it in the UI. Do you currently have a use for scheduled_end_time? should it be the same as end_time initially?

Reply:
We have a use case for `scheduled_end_time` - when a user taps a "Stop experiment" button we will want to update the end_time and leave `scheduled_end_time` unchanged - both end_time and scheduled_end_time will be displayed in experiment details view (which is going to be merged into the main 'soon').

Reply:
`creation_time` will be displayed in experiment details view too.

Reply:
nvm, we can address this in a separate PR too - we use `ExperimentRunDetails` when we display creation_time and scheduled_end_time in the UI so the changes that I requested are not a must here.